openvdb/tree/RootNode.h

```diff
@@ -1546,6 +1547,14 @@ RootNode<ChildT>::nonLeafCount() const
 }
 
 
+template<typename ChildT>
+inline Index32
+RootNode<ChildT>::childCount() const
+{
+    return this->getChildCount();
+}
```

Comment:
As I understand it, this is required due to making private methods public changing ABI on Windows? With this, can we remove `getChildCount()` entirely?

Reply:
Yes, it looks like it's only being called by "writeTopology()" so that's an easy change. 

Incidentally, I briefly looked at trying to unify the rather confusing set of count methods - for example:

* RootNode::getActiveTileCount() - counts all the active tiles in the root node and **not** the nodes below
* InternalNode::onTileCount() - counts all the active tiles in this internal node **and** all nodes below

It's not totally clear what to do here though, so I think I'll pass on attempting this at least until we introduce a tools/Count.h to put some threaded, recursive variations.

Reply:
As removing it will break ABI as well (*?) I suggest updated all usage and wrapping the old one in an ifdef ABI 8 clause

* https://community.kde.org/Policies/Binary_Compatibility_Issues_With_C%2B%2B#The_Do.27s_and_Don.27ts

Reply:
Actually sorry, this is fine so long as "they're never called from an inline method"

Reply:
Ah yes, the writeTopology method is marked inline, so we can't safely remove it.

There's no ABI=8 defined yet and we agreed a while ago not to define ABI _beyond_ the current ABI. Instead, I've created a JIRA with the "ABI8" label - https://jira.aswf.io/issues/?jql=labels%20%3D%20ABI8

Reply:
> Actually sorry, this is fine so long as "they're never called from an inline method"

It's not totally clear to me, but writeTopology _is_ inline, so think it's safest to just punt this change until later.

Reply:
Well if there's ever any doubt we'd need to wrap it in an compile time define between major versions? Could do that now?